### DISCUSSION TITLE
Include labels and milestone after the issue title

### DISCUSSION THREAD

**Discussion by egvijayanand** [community/community#147655] [Mobile]
Include labels and milestones, if any, after the issue title.

It can be displayed alongside the issue status.

**Comment by aconsuegra** [comment]
Thanks for your feedback @egvijayanand! We are displaying all the metadata (labels, milestones, etc.) below the issue title is issues lists. In which screen would you like to see metadata after the issue title?

**Reply by egvijayanand** [reply]
This feedback pertains to the GitHub mobile app.

I would like to view those metadata on the Issue details screen.
